app/grants/models.py

```diff
@@ -900,3 +900,25 @@ class MatchPledge(SuperModel):
     def __str__(self):
         """Return the string representation of this object."""
         return f"{self.profile} <> {self.amount} DAI"
+
+class PhantomFunding(SuperModel):
+    """Define the structure of a subscription agreement."""
```

Comment:
Can we add something that describes what this is for or why we created it? Looks like this is from a cut-and-paste job ;)